implementation/leprechaun/src/vehicle/controller/vehicle.dto.ts

```diff
@@ -0,0 +1,43 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Vehicle } from '../database/vehicle.entity';
+
+export class VehicleDto {
+  @ApiProperty({
+    description: 'Vehicle ID',
+    maxLength: 36,
+    minLength: 36,
+    nullable: false,
+    examples: ['f1b1b9b1-1c1b-4b1b-9b1b-1c1b1b1b1b1b'],
```

Comment:
**question:** Does `format: "uuid"` work here?

Reply:
```ts
@ApiProperty({
  format: 'uuid',
  description: 'Vehicle ID',
  nullable: false,
})
```
![obraz](https://github.com/user-attachments/assets/9433c91b-670c-46a5-b48c-4c0cbbe23c9e)

```ts
@ApiProperty({
  description: 'Vehicle ID',
  maxLength: 36,
  minLength: 36,
  nullable: false,
  examples: ['f1b1b9b1-1c1b-4b1b-9b1b-1c1b1b1b1b1b'],
})
```
![obraz](https://github.com/user-attachments/assets/df6b2f72-6af7-41f3-91ca-484feaaad2ff)

Neither of them look proper to me.

Reply:
I'd combine `format: "uuid"` and your own `examples`.